### PR TITLE
Slider alcohol

### PR DESCRIPTION
**Pull request by c3soft**
fixed slider

### CODE REVIEW

File: src/components/AlcoholFilter.js [hunk 1]
```diff
@@ -43,47 +44,42 @@ class AlcoholFilter extends React.Component {
...
+					/>
+				</div> */}
+				{/* <div className="sweetAndSaltyButtons">
+					<button className="Sucré" onClick={this.saltyOrSweet}>
+						Sucré
+					</button>
+					<button className="Salé" onClick={this.saltyOrSweet}>
+						Salé
+					</button>
+				</div> */}
```
Comment (erabin05) [inline src/components/AlcoholFilter.js]: IF YOU DON'T USE SOME CODE REMOVE IT

File: src/components/Sliders/SliderAlcohol.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        max: 0,
+        
+      },
+      choiseAlcohol: true,
+        choiseNo: true,
+    };
+  }
+
+  showSliders=()=>{
+    // style={display: this.state.choiseAlcohol ? 'none' : 'block' }
```
Comment (erabin05) [inline src/components/Sliders/SliderAlcohol.js]: SAME

File: src/components/Sliders/SliderAlcohol.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      this.setState({
+        
+        choiseAlcohol : true,
+          choiseNo : false
+      })
+    }
+    showNoAlcohol=()=>{
+      // style={display: this.state.choiseAlcohol ? 'none' : 'block' }
```
Comment (erabin05) [inline src/components/Sliders/SliderAlcohol.js]: SAME

File: src/components/Sliders/SliderAlcohol.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+//import ReactDOM from 'react-dom';
```
Comment (erabin05) [inline src/components/Sliders/SliderAlcohol.js]: SAME

File: src/components/Sliders/slider.css [hunk 5]
```diff
@@ -0,0 +1,51 @@
+.formSlider{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  /* height: 80%; */
+}
+/* .showSlider{
+  display: none;
+} */
```
Comment (erabin05) [inline src/components/Sliders/slider.css]: SAME

**Review by erabin05 — CHANGES_REQUESTED** [review]
Remove the code you don't use, don't put it as comment !!
If you are afraid to loose code you made, just commit regularly

**Comment by Cindie1908** [comment]
It's done

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**